### ISSUE TITLE
Very long replays don't show up in the Replay Viewer

### ISSUE THREAD

**Issue by ItsTinay** [ReplayMod/ReplayMod#487]

**Comment by Johni0702** [comment]
Yes they do (as long as "very long" is defined as less than 30 days).

If they do not for you, you will have to provide more information.

**Comment by ItsTinay** [comment]
What I meant is I was recording for 2-3 hours and after saving the replay it didn't show up in the Replay Viewer (The replay's size is 330 MB)

**Comment by ItsTinay** [comment]
May some other mods I installed be the cause of this issue?

**Comment by ItsTinay** [comment]
Tried Replay Mod 2.5.2 with no extra mods, my recording is still not showing in the Replay Viewer (but it IS in the replay_recordings folder)
